server/dbSchemas/attendanceSchema.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const userSchema = new Schema({
+    email: { type: String, required: true, trim: true },
+    name: { type: String, required: true, trim: true },
+    password: { type: String, required: true },
+    is_student: { type: Boolean, required: true },
+    is_professor: { type: Boolean, required: true },
+    student_number: String
+});
+const User = mongoose.model('User', userSchema, 'user');
+
+const courseSchema = new Schema({
+    name: { type: String, required: true, trim: true },
+    admin: { type: mongoose.Schema.Types.ObjectId, ref: 'user' },
+    date_created: { type: Date, required: true }
+});
+const Course = mongoose.model('Course', courseSchema, 'course');
+
+const sectionSchema = new Schema({
```

Comment:
So Dawn mentioned that sections should be distinguished as separate courses. I would propose to merge the "Section" and "Course" together as one "Course". Using that model every section for a class will be created as separate courses in the application, which would look like "SWE4103_FR01A", "SWE4103_FR01B". Another reason for merging both together is that all the data we need to store is per section, not per course.